test/fixtures/output/php/guzzle/custom-method.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+$client = new \GuzzleHttp\Client();
+
+$response = $client->request('PROPFIND', 'http://mockbin.com/har', [
```

Comment:
Can you update the target to only add the third param if it's got content? Should clean this example up a touch.

Reply:
@robertoarruda Sorry, could you update the Guzzle target as well with this change? Tests are currently failing because they're generating snapshots that match the old style.